### ISSUE TITLE
Have the UWG output an Error Report so that people can u derstand when a simulation goes wrong

### ISSUE THREAD

**Issue by chriswmackey** [hansukyang/UWG_Matlab#3]
It would be great to get an error report text file out of the UWG if it fails so that we can easily report the problem to the user and tell them what to change.  EnergyPlus does this and it has been essential to my workflows with it (I have the grasshopper component give warnings if EnergyPlus encounters any errors).

**Comment by hansukyang** [comment]
Yes, so at the moment, the plan is to create a log file and we can have a section with error logs there. At the moment, it's probably easy to do a quick check on the parameters to make sure that they are not faulty.
